### DISCUSSION TITLE
编译 sjtureport 显示 ! LaTeX Error: The key 'sjtu/info/zh/title' is unknown and is being ignored.

### DISCUSSION THREAD

**Discussion by CrazyElements** [sjtug/SJTUThesis#927] [提问] [answered]
windows11, telive 2023, VS Code 

编译 `\documentclass{sjtuthesis}` 没有问题，但编译 `\documentclass{sjtureport}`  和 `\documentclass{sjturarticle}` 时报错 ! LaTeX Error: The key 'sjtu/info/zh/title' is unknown and is being ignored。
看起来像是 `setup.tex` 和`sjtureport.cls`的配置问题
就是在这附近
![image](https://user-images.githubusercontent.com/105211051/232710245-81e6a5c6-4dab-4dc7-aaec-9b2a6d77f7b0.png)
![image](https://user-images.githubusercontent.com/105211051/232710949-10418ba5-f2af-4867-8e95-898f83342fcf.png)

**Comment by stone-zeng** [comment, marked as answer]
在 `sjtureport` 里面 `title` 等字段没有 `zh/`

**Reply by CrazyElements** [reply]
谢谢你，现在可以使用了😊

**Comment by CrazyElements** [comment]
谢谢你，现在可以使用了😊
